src/Lecture1.hs

```diff
@@ -119,4 +139,8 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String
+-- I guess @count@ is implemented recursively, does that count? Hope so, I made it with love 💜
+lowerAndGreater n list =
+    show n ++ " is greater than " ++ show (count (< n) list) ++ " elements and lower than " ++ show (count (> n) list) ++ " elements"
```

Comment:
That's correct! 💯 

However, with this solution, you will walk through the list twice.
You can write it more performant with only one traversal of the list 🙂
It's desirable to implement functions that traverse lists a single time and consume each element not more than once. In that case, the composition of such functions will have nice streaming and lazy properties 🦥

Reply:
ah right, laziness :) I guess I could just build a single tuple with min- and max-count in a fold?

Reply:
You don't need a tuple or a fold. You can implement a manual recursive function that takes two `Int`s and outputs the resulting string in the end.